public/js/login.js

```diff
@@ -2,14 +2,14 @@
 $('#login').submit(function(evt) {
     evt.preventDefault();
     $.ajax({
-        type: 'POST',
+        type: 'GET',
```

Comment:
Logins requests are generally supposed to be POST requests so the username and password go through the body and not the URL. Is there a specific reason why we changed it to a GET?

Reply:
https://stackoverflow.com/questions/630453/put-vs-post-in-rest

Put and post are used to update and create resources on the backend. When you login, you don't add to the backend, you just fetch

Reply:
Technically you're "creating" a session in the backend when you're logging in (even though we don't have sessions). POST give a more secure way of executing a login and it not only keeps the URL clean looking, it prevents your salted and hashed password from being starkly visible in the URL, prevents against browser prefetching, and prevents against search engine spidering. See the following:

https://stackoverflow.com/questions/15098392/which-http-method-should-login-and-logout-actions-use-in-a-restful-setup

https://stackoverflow.com/questions/3521290/logout-get-or-post/14587231#14587231

https://stackoverflow.com/questions/7140074/restfully-design-login-or-register-resources

https://stackoverflow.com/questions/5868786/what-method-should-i-use-for-a-login-authentication-request

https://stackoverflow.com/questions/2001773/understanding-rest-verbs-error-codes-and-authentication

Reply:
As we discussed offline, it's best to use post. Will fix